In [1]:
import pandas as pd

In [2]:
url_csv = 'https://raw.githubusercontent.com/plotly/datasets/master/iris-data.csv'
df = pd.read_csv(url_csv)
print(df.shape)
print(df.head())

(150, 5)
   sepal length  sepal width  petal length  petal width        class
0           5.1          3.5           1.4          0.2  Iris-setosa
1           4.9          3.0           1.4          0.2  Iris-setosa
2           4.7          3.2           1.3          0.2  Iris-setosa
3           4.6          3.1           1.5          0.2  Iris-setosa
4           5.0          3.6           1.4          0.2  Iris-setosa


### Descripción del dataset:
El dataset contiene 150 filas y 5 columnas. Las columnas son: sepal_length, sepal_width, petal_length, petal_width y species. Estas representan las medidas de longitud y ancho de los sépalos y pétalos de las flores de iris, y la especie de la flor, respectivamente.

In [3]:
print(df.isnull().sum())


sepal length    0
sepal width     0
petal length    0
petal width     0
class           0
dtype: int64


### Presencia de valores nulos:
El dataset no presenta valores nulos en ninguna de sus columnas.

In [4]:
print(df.describe())

       sepal length  sepal width  petal length  petal width
count    150.000000   150.000000    150.000000   150.000000
mean       5.843333     3.054000      3.758667     1.198667
std        0.828066     0.433594      1.764420     0.763161
min        4.300000     2.000000      1.000000     0.100000
25%        5.100000     2.800000      1.600000     0.300000
50%        5.800000     3.000000      4.350000     1.300000
75%        6.400000     3.300000      5.100000     1.800000
max        7.900000     4.400000      6.900000     2.500000
